tests/test_base.py

```diff
@@ -168,3 +169,22 @@ def test_config_set(self):
                 DefFeature("Year", int, 1), DefFeature("Commits", int, 10)
             ),
         )
+
+@config
+class FakeTestingConfig2:
+    num: float
+    name: str = field("Name of FakeTesting2")
+    features: Features = Features(
+                DefFeature("default", int, 1), DefFeature("features", int, 10)
+            )
+    label: str = "unlabeled"
+
+@entrypoint("fake2")
+class FakeTesting2(BaseTesting):
+    
+    CONFIG = FakeTestingConfig2
+
+class TestCONFIG(unittest.TestCase):
+    def test_CONFIG(self):
+        with self.assertRaises(MissingConfig):
```

Comment:
```suggestion
        with self.assertRaisesRegex(TypeError, "__init__() missing 1 required positional argument: 'config'"):
```

Reply:
I just did this because it threw that error at me :D Not getting the TypeError yet. Gonna try regular instantiation.

Reply:
@pdxjohnny Also how do I set the default value right after this test to test the successful case?

Reply:
Probably best to just make another class 

```python
@config
class FakeTestingConfig3:
    label: str = "unlabeled"

@entrypoint("fake3")
class FakeTesting3(BaseTesting):

    CONFIG = FakeTestingConfig3
```

Reply:
Yeah crossed my mind, was reluctant. Thanks!
`with self.assertRaises(TypeError):` working with the normal instantiation now. `self.assertRaisesRegex` throws assertion error. 
Pushing in a bit.